mate-panel/libmate-panel-applet-private/panel-applets-manager-dbus.c

```diff
@@ -163,6 +164,14 @@ mate_panel_applets_manager_get_applet_factory_info_from_file (const gchar *filen
 
 			return NULL;
 		}
+		lib_prefix = g_getenv ("MATE_PANEL_APPLET_LIB_PREFIX");
+		if (lib_prefix && g_strcmp0 (lib_prefix, "") != 0) {
+			char *location = g_strnfill (strlen (lib_prefix) + strlen (info->location), 0);
+			strcat (location, lib_prefix);
+			strcat (location, info->location);
```

Comment:
This give us a new warning
```
panel-applets-manager-dbus.c:170:4: warning: Call to function 'strcat' is insecure as it does not provide bounding of the memory buffer. Replace unbounded copy functions with analogous functions that support length arguments such as 'strlcat'. CWE-119
                        strcat (location, lib_prefix);
                        ^~~~~~
panel-applets-manager-dbus.c:171:4: warning: Call to function 'strcat' is insecure as it does not provide bounding of the memory buffer. Replace unbounded copy functions with analogous functions that support length arguments such as 'strlcat'. CWE-119
                        strcat (location, info->location);
                        ^~~~~~
2 warnings generated.
```
https://travis-ci.org/mate-desktop/mate-panel/jobs/546928776#L1661

Reply:
Should be fixed